vtex-sites/nextjs.store
# Feat: Refinements on `Navbar` and `Navlinks`

## What's the purpose of this pull request?
This PR follows new styles guidelines for `Navbar` and `Navlinks`

## How does it work?
Additionally to visual changes, we added a scroll interaction on desktop:

https://user-images.githubusercontent.com/11613011/182395277-1ff46d43-241d-4ef7-853c-905e7846b114.mov

## How to test it?
You can check the [Navbar](https://nextjs-store-storybook-git-feat-refinements-he-78a5d3-faststore.vercel.app/?path=/docs/organisms-navbar-overview--overview-default) and [Navlink](https://nextjs-store-storybook-git-feat-refinements-he-78a5d3-faststore.vercel.app/?path=/docs/organisms-navbar-navlinks--overview-default) stories.  You can also check the scroll effect at the [Demo Store](https://preview-191--nextjs.preview.vtex.app/).

## References
https://www.codemzy.com/blog/react-sticky-header-disappear-scroll

## Checklist
**PR Title and Commit Messages**

**PR Description**

**Documentation**
